hellabot.go

```diff
@@ -270,11 +270,20 @@ func (bot *Bot) Reply(m *Message, text string) {
 
 // Msg sends a message to 'who' (user or channel)
 func (bot *Bot) Msg(who, text string) {
-	for len(text) > 400 {
-		bot.Send("PRIVMSG " + who + " :" + text[:400])
-		text = text[400:]
+	scanner := bufio.NewScanner(strings.NewReader(text))
+	for scanner.Scan() {
+		bot.sendline(who, scanner.Text())
+	}
+}
```

Comment:
Pull the splitting logic into a separate function which handles both newlines and long lines. Then re-use that function in the `Notice` call.

``` go
func splitText(s string) []string
```

Reply:
That function is used multiple times, shouldn't it be placed into a separate `utils.go` file ?

Reply:
Doesn't matter, I wouldn't bother.

Reply:
Thanks.